### PR TITLE
prepare 3.3.8 release

### PR DESCRIPTION
**Pull request by errotan**

### CODE REVIEW

File: system/classes/Kohana/Core.php [hunk 1]
```diff
@@ -16,8 +16,8 @@
 class Kohana_Core {
 
 	// Release version and codename
-	const VERSION  = '3.3.7';
-	const CODENAME = 'lazarus';
+	const VERSION  = '3.3.8';
+	const CODENAME = 'cubelles';
```
Comment (piotrbaczek) [inline system/classes/Kohana/Core.php]: Cool, has there been any name voting? 👍

Comment (errotan) [reply system/classes/Kohana/Core.php]: Yes, on telegram.

**Comment by errotan** [comment]
Will merge and tag tomorrow night if nobody protests.

**Comment by neo22s** [comment]
All looking good :D

**Pull request merged**